src/client/sky.cpp

```diff
@@ -605,7 +471,8 @@ void Sky::update(float time_of_day, float time_brightness,
 	// pure white: becomes "diffuse light component" for clouds
 	video::SColorf cloudcolor_bright_normal_f = video::SColor(255, 255, 255, 255);
 	// dawn-factoring version of pure white (note: R is above 1.0)
-	video::SColorf cloudcolor_bright_dawn_f(255.0f/240.0f, 223.0f/240.0f, 191.0f/255.0f);
+	video::SColorf cloudcolor_bright_dawn_f(
+			255.0f / 240.0f, 223.0f / 240.0f, 191.0f / 255.0f);
```

Comment:
Please only wrap lines at 90 columns (tab size 4). This may be a bad clang format suggestion.